runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -238,6 +245,31 @@ public CorfuRuntime setMetrics(@NonNull MetricRegistry metrics) {
         return this;
     }
 
+    /**
+     * These two handlers are provided to give some control on what happen when system is down.
+     *
+     * For applications that want to have specific behaviour when a the system appears unavailable, they can
+     * register their own handler for both before the rpc request and upon network exception.
+     *
+     * An example of how to use these handlers implementing timeout is given in
+     * test/src/test/java/org/corfudb/runtime/CorfuRuntimeTest.java
+     *
+     */
+    public Runnable beforeRpcHandler = () -> {};
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make beforeRpcHandler a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)